### DISCUSSION TITLE
Massiv Spam Problem

### DISCUSSION THREAD

**Discussion by Serverfrog** [community/community#45358] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

In the last 2 Months it seems that Github has a Problem with Fake Profiles created to distribute Trojan Horses in Password Protected Archives. These are disguised as Cheats/Hacks/cracks for popular Games/Software.

The Problem i have, i have a Limit how many users i can report, they don't have a Limit how many they create.

I use githunt as Browser Startpage, and its full of Fake Repositories that are Starred up with a massive amounts of either freshly created Accounts or (i suspect) hacked accounts that also sometimes have more than 5k stargazes.

What is hard task to have a cronjob that looks all 10 minutes or 1 hour at the top stargazed projects on that day and check them? I mean it is also needed to check the Reports so its not more manpower that is needed. And with the technology knowledge behind Github it should also be possible to automatically detected these kinds of Repos.

If you dont know what im Talking of:

  - https://github.com/wangzqe2/Updated-24.01.2023-Hack-for-Dota-2-MapsHack-COMBO-SKINCHANGER-
  - https://github.com/HuYaTVLM/Updated-24.01.2023-DBD-Big-functions
  - https://github.com/neness16/Updated-24.01.2023-HACK-TF-2-AIM-ESP-TRIGGERBOT-BHOP-JUMP
  - https://github.com/SQ-sanqi/Updated-24.01.2023-Hack-for-DayZ-AIM-ESP-LOOT-MISC
  - https://github.com/anthonydasilva5678/Fortnite-Hack-Undetected-Cheat-External

**DON'T DOWNLOAD THE ARCHIVES**

**Comment by jge162** [comment]
Thanks for the heads up.

**Comment by JohanAR** [comment]
We're also having spam problems, except with people replying to all open issues and posting their whatsapp number, claiming they have a solution (that it turns out that they want $200-500 for) or that they need to connect to the user's computer using anydesk/teamviewer.

Not sure if I ought to delete all the posts after reporting someone, or hide it and flag as spam so that github staff sees it if they ever review the report.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there folks 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
